pom.xml

```diff
@@ -36,7 +36,7 @@
         <version.org.wildfly.core>10.0.3.Final</version.org.wildfly.core>
         <version.org.wildfly.extras.creaper>1.6.1</version.org.wildfly.extras.creaper>
         <version.org.yaml.snakeyaml>1.24</version.org.yaml.snakeyaml>
-        <version.com.fasterxml.jackson.core.jackson-databind>2.9.10.1</version.com.fasterxml.jackson.core.jackson-databind>
+        <version.com.google.code.gson>2.8.5</version.com.google.code.gson>
```

Comment:
There is `2.8.6` already, could you update it, please?

Reply:
2.8.6 failed several (2 or 3) attempts for Travis checks yesterday on jdk11. Then I bumped it to 2.8.5: https://github.com/jboss-eap-qe/eap-microprofile-test-suite/pull/88#issuecomment-564674003

Reply:
Looks like it was cause by network issue in travis when downloading it. Version was updated and PR merged.